Xamarin.Forms.Platform.iOS/EventTracker.cs

```diff
@@ -279,22 +301,62 @@ UITapGestureRecognizer CreateTapRecognizer(int numTaps, Action action, int numFi
 				ShouldRecognizeSimultaneously = ShouldRecognizeTapsTogether
 			};
 
+			return result;
+		}		
+#else
+		NativeGestureRecognizer CreateClickRecognizer(int buttonMask, int numberOfClicksRequired, Action returnAction)
```

Comment:
why are we moving this code?

Reply:
I think its more to do with the ShouldRecognizeTapsTogether pushes it down.  It used to be in the #if __MOBILE__ else statement but now is used for the macOS side.